cmd/chartsvc/handler.go

```diff
@@ -61,34 +63,127 @@ type rel struct {
 	Links selfLink    `json:"links"`
 }
 
-// listCharts returns a list of charts
-func listCharts(w http.ResponseWriter, req *http.Request) {
+type meta struct {
+	TotalPages int `json:"totalPages"`
+}
+
+type count struct {
```

Comment:
maybe add a comment to say that this is used to retrieve a count from the database (might seem obvious, but it's not so obvious that we need a struct for this rather than just a variable)